### PR TITLE
Fix for warnings from world builder.

### PR DESCRIPTION
**Pull request by MFraters**
@bobmyhill and @anne-glerum could you test if this fixes the issue?

@tjhei It would be great to take this over in the release branch

### CODE REVIEW

**Comment by tjhei** [comment]
This makes sense. Are these the warnings I reported to you?
I am surprised you want to fix things directly. Why not make a point release of WB and import the new version here? Or is that too much effort?

**Comment by MFraters** [comment]
When and where did you send them? These where warnings reported by @anne-glerum  and @bobmyhill.

I also made a pull request for the world builder in https://github.com/GeodynamicWorldBuilder/WorldBuilder/pull/503. I could make a point release, I am not sure if it is worth it.

File: contrib/world_builder/source/world_builder/world.cc [hunk 1]
```diff
@@ -304,6 +304,7 @@ namespace WorldBuilder
     // We receive the cartesian points from the user.
     const Point<3> point(point_,cartesian);
 
+    (void) this->limit_debug_consistency_checks;
     WBAssert(!this->limit_debug_consistency_checks || this->parameters.coordinate_system->natural_coordinate_system() == cartesian
```
Comment (bangerth) [inline contrib/world_builder/source/world_builder/world.cc:308]: Does it really warn about this variable being unused? It's a member variable, how does the compiler know that the function isn't used anywhere else?

Comment (MFraters) [reply contrib/world_builder/source/world_builder/world.cc:308]: In debug mode is isn't used anywhere. I never got this warning, but it does appear on some mac's apparently.

Comment (bangerth) [reply contrib/world_builder/source/world_builder/world.cc:308]: Fascinating.

**Comment by bangerth** [comment]
I'm ok to merge as-is, in the same way as we often apply small patches to bundled packages to avoid warnings. But I'll defer to @tjhei if he has a different opinion.

**Review by tjhei — APPROVED** [review]
Fine with me either way.

**Comment by tjhei** [comment]
Here is the issue I opened: https://github.com/GeodynamicWorldBuilder/WorldBuilder/issues/502

**Comment by MFraters** [comment]
> Here is the issue I opened: [GeodynamicWorldBuilder/WorldBuilder#502](https://github.com/GeodynamicWorldBuilder/WorldBuilder/issues/502)

Ah, sorry. I had completely missed that. Can you confirm that it is fixed now?

**Comment by MFraters** [comment]
@bobmyhill confirmed that the warnings are gone for him.

**Pull request merged**